AirCasting/Notes/EditNote/EditNoteView.swift

```diff
@@ -36,6 +40,16 @@ private extension EditNoteView {
         createNoteTextField(binding: $viewModel.noteText)
     }
     
+    var photo: some View {
+        HStack {
+            if let url = viewModel.notePhoto {
+                Spacer()
+                DownloadableImage(url: url)
+                Spacer()
```

Comment:
I think you can use here maybe:

`
DownloadableImage(url: url)
  .frame(maxWidth: .infinity, alignment: .center)`